jobs/delorean/jenkinsfiles/discovery/ga/github/Jenkinsfile

```diff
@@ -39,46 +27,95 @@ static String getRHMIReleaseBranch(releaseTag) {
     return "v${majorVersion}.${minorVersion}"
 }
 
-boolean doUpdateForLevels(product, current, latest, levels) {
-    def utils = new org.integr8ly.Utils()
-    String[] currentSemVer = utils.extractSemVer(current, product)
-    String[] latestSemVer = utils.extractSemVer(latest, product)
-
-    boolean isMajorUpdate = latestSemVer[0].toInteger() > currentSemVer[0].toInteger()
-    boolean isMinorUpdate = latestSemVer[1].toInteger() > currentSemVer[1].toInteger()
-    int latestPatch = 0
-    int currentPatch = 0
-    if (currentSemVer.size() > 2) {
-        currentPatch = currentSemVer[2].toInteger()
+String lookupProductAnsibleRole(String product) {
+    String role = product
+    switch (product) {
+        case "amq-online":
+            role = "enmasse"
+    }
+    return role
+}
+
+String[] extractSemVer(String productVersion, String productName) {
+    if (productName == "fuse") {
+        List parts = productVersion.replaceAll("[^0-9/-]", "").tokenize("-")
+        return [parts[1].substring(0, 1), parts[1].substring(1, 2), parts[1].substring(2, 6), parts[2]]
+    } else {
+        return productVersion.replaceAll("[^0-9/.]", "").tokenize(".")
     }
-    if (latestSemVer.size() > 2) {
-        latestPatch = latestSemVer[2].toInteger()
+}
+
+boolean verifyIsGA(String product, String version) {
+    if (product == "fuse-online") {
+        //ToDo Since fuse online have changed their installation we have currently have no way of knowing if it's GA or not.
+        return version < "1.8.0"
```

Comment:
Need a follow on Jira here since this is obviously a temporary workaround and will fail discover the correct version once 1.8  is GA'd.